src/ui/page_imagesettings.c

```diff
@@ -52,8 +53,9 @@ static lv_obj_t *page_imagesettings_create(lv_obj_t *parent, panel_arr_t *arr) {
     create_slider_item(&slider_group2, cont, "Saturation", 47, g_setting.image.saturation, 2);
     create_slider_item(&slider_group3, cont, "Contrast", 47, g_setting.image.contrast, 3);
     create_slider_item(&slider_group4, cont, "OLED Auto off", 3, g_setting.image.auto_off, 4);
+    create_slider_item(&slider_group5, cont, "Embedded OSD Mode", 2, g_setting.osd.embedded_mode, 5);
```

Comment:
I think the `2` should be `1` as the function uses that as the `max` value with `0` being the minimum.
So on the main display when it's 16x9 the slider is in the middle!